### PR TITLE
Add toast to inform the user that download started.

### LINKED ISSUE #5428: Toast notification when download started
### Checklist

- [x] I checked, but didn't find any duplicates (open OR closed) of this issue in the repo. 
- [ x] I have read the contribution guidelines given at https://github.com/TeamNewPipe/NewPipe/blob/HEAD/.github/CONTRIBUTING.md.
- [ x] This issue contains only one feature request. I will open one issue for every feature I want to request.

#### Describe the feature you want

I think having a toast notification for when a download has started would be useful. Sometimes I wonder if I clicked correctly or just closed the download prompt without having it started, and then I have to go to downloads screen to double-check... That's it! :)

#### Is your feature request related to a problem? Please describe it

No

#### How will you/everyone benefit from this feature?

I think toast notifications are quite widespread and standard and users maybe used to them or waiting for them...

### PR DESCRIPTION
**Pull request by WoodyMats**
### **What is it?**

- [ ] Bugfix (user facing)
- [x]  Feature (user facing)
- [ ]  Codebase improvement (dev facing)
- [ ] Meta improvement to the project (dev facing)

### **Description of the changes in your PR**

- Add toast to inform the user that download started
- Add the right string in values folder
- fixes #5428 

### **Due diligence**

- [x] I read the [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/HEAD/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by WoodyMats** [comment]
#5428 = Linked issue.

**Comment by mhmdanas** [comment]
@WoodyMats thanks for opening a PR! Could you please not remove the template next time you open a PR?

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -700,4 +700,5 @@
     <string name="recent">Recent</string>
     <string name="chapters">Chapters</string>
     <string name="no_app_to_open_intent">No app on your device can open this</string>
+    <string name="download_has_started">Download has started</string>
```
Comment (mhmdanas) [inline app/src/main/res/values/strings.xml:703]: Maybe it should be `Downloading $VIDEO_NAME` instead.

**Comment by XiangRongLin** [comment]
Please don't open the PR against the release branch. It needs to be dev branch

File: app/src/main/res/values/strings.xml (hunk 1 above)
Comment (XiangRongLin) [reply app/src/main/res/values/strings.xml:703]: I don't see it as needed. The issue does not ask for this.

Comment (WoodyMats) [reply app/src/main/res/values/strings.xml:703]: > @WoodyMats thanks for opening a PR! Could you please not remove the template next time you open a PR?

Sorry! Now i just see the PR template. In future PRs i will do it with the right way.

Comment (WoodyMats) [reply app/src/main/res/values/strings.xml:703]: Can you explain me the issue so i can do it with the right way? :)

**Review by mhmdanas — CHANGES_REQUESTED** [review]
Since your PR was initially opened against the release branch and then changed to the development branch, you still have a commit from the release branch (the commit that changes `app/build.gradle`). Could you revert that commit?

**Comment by WoodyMats** [comment]
> Since your PR was initially opened against the release branch and then changed to the development branch, you still have a commit from the release branch (the commit that changes `app/build.gradle`). Could you revert that commit?

May i also add the Video name as you said in previous comment?

**Comment by mhmdanas** [comment]
There isn't consensus on that, so you probably shouldn't add it for now.

**Comment by WoodyMats** [comment]
> There isn't consensus on that, so you probably shouldn't add it for now.

Okk! The requested change just done. 😃

File: app/src/main/java/org/schabi/newpipe/download/DownloadDialog.java [hunk 2]
```diff
@@ -684,6 +684,9 @@ private void prepareSelectedDownload() {
         prefs.edit()
                 .putString(getString(R.string.last_used_download_type), selectedMediaType)
                 .apply();
+
+        Toast.makeText(context, getString(R.string.download_has_started),
+                Toast.LENGTH_LONG).show();
```
Comment (XiangRongLin) [inline app/src/main/java/org/schabi/newpipe/download/DownloadDialog.java:689]: This is something someone with a better eye for UX can answer, but i don't think `LENGTH_LONG` (3.5 seconds) is necessary. `LENGTH_SHORT` (2 seconds) should be enough in my view.

https://stackoverflow.com/questions/7965135/what-is-the-duration-of-a-toast-length-long-and-length-short

**Review by XiangRongLin — COMMENTED** [review]
Other than that get the attention of the guy who opened the issue to test out the changes please.

Edit: and others who have shown interest in the issue being resolved

File: app/src/main/java/org/schabi/newpipe/download/DownloadDialog.java (hunk 2 above)
Comment (WoodyMats) [reply app/src/main/java/org/schabi/newpipe/download/DownloadDialog.java:689]: You are right. I tried both and i think that `LENGTH_SHORT` looks more consistent. I'm changing it now.

**Comment by WoodyMats** [comment]
Good morning! 😄 I have changed the toast notification's duration from `LENGTH_LONG` to `LENGTH_SHORT`. You can check it! @mhmdanas

**Comment by XiangRongLin** [comment]
> Other than that get the attention of the guy who opened the issue to test out the changes please.
> 
> Edit: and others who have shown interest in the issue being resolved

@WoodyMats see my last comment

**Comment by Stypox** [comment]
@XiangRongLin is there anything preventing this commit from being merged?

**Comment by XiangRongLin** [comment]
@Stypox Codewise nothing. As i said in my last comment it would be good to let the issue opener test out the changes. To see if it solves the problem for them.

**Comment by Stypox** [comment]
@illustribe could you test, please?

**Comment by illustribe** [comment]
@stypox I've tested in version 0.20.11 and there's no toast message when one starts a download 🤷‍♂️ both while downloading audio or video...

**Comment by mhmdanas** [comment]
@illustribe you're supposed to test this PR's APK, as this change has still not been incorporated into 0.20.11. Here's [the link](https://github.com/TeamNewPipe/NewPipe/suites/1964180840/artifacts/38976161).

**Comment by illustribe** [comment]
@Stypox it's working well! 👍

**Comment by Stypox** [comment]
@WoodyMats @illustribe thank you!

**Pull request merged**